src/foam/dao/HTTPSink.java

```diff
@@ -64,12 +62,12 @@ public void put(Object obj, Detachable sub) {
       }
       conn.connect();
 
-      os = conn.getOutputStream();
-      writer = new BufferedWriter(new OutputStreamWriter(os, StandardCharsets.UTF_8));
-      writer.write(outputter.stringify((FObject)obj));
-      writer.flush();
-      writer.close();
-      os.close();
+      try(OutputStream os = conn.getOutputStream()) {
```

Comment:
Add space after 'try', this line and next.

Reply:
fixed